Automattic/wp-calypso
# Domains Management Revamp: Add breadcrumbs to domain sub-pages

Related to #96832
Based on https://github.com/Automattic/wp-calypso/pull/97562

## Proposed Changes

* Add breadcrumbs to the sub-pages.
* The breadcrumbs are only 1 level deep for now. We can extend it when we have the screens that need it.

## Why are these changes being made?


This is part of the Domain Management Revamp project: pfuQfP-13x-p2

## Testing Instructions



* Checkout the branch.
* Go to http://calypso.localhost:3000/domains/manage?flags=calypso/all-domain-management.
* Click on a domain where you can edit the contact info.
* Go to the "Contact Information" section and click the "Edit" button.
* You will be redirected to http://calypso.localhost:3000/domains/manage/all/contact-info/edit/[site]/[domain]
* Compare the page with the design - ePfz53j71KKDLWtuSm567A-fi-4949_49077. Note that the header is slightly different from the design as explained [here](https://github.com/Automattic/wp-calypso/pull/97368).
* Check that the page looks good on mobile.

<img width="984" alt="image" src="https://github.com/user-attachments/assets/9e25bfa5-3915-4832-9b1c-c255c020f7ff" />

## Pre-merge Checklist
